src/main/java/mate/academy/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,101 @@
+package mate.academy.shop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.apache.log4j.Logger;
+import mate.academy.shop.dao.ProductDao;
+import mate.academy.shop.lib.Dao;
+import mate.academy.shop.model.Product;
+import mate.academy.shop.util.ConnectionUtil;
+import mate.academy.shop.web.filters.AuthorizationFilter;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    public static final Logger logger = Logger.getLogger(ProductDaoJdbcImpl.class);
+
+    @Override
+    public Product create(Product product) {
+        String query = "INSERT INTO products (product_name, price) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setString(1, product.getName());
+            statement.setDouble(2, product.getPrice());
+            statement.execute();
+        } catch (SQLException e) {
+            logger.error("Error at the adding product to db!");
```

Comment:
do not log errors before throwing the same exception. This is a duplication of info